testsuite/meta/binary/newton_support_vector_machine.dat

```diff
@@ -1,12 +1,12 @@
 <<_SHOGUN_SERIALIZABLE_ASCII_FILE_V_00_>>
 array Vector<SGSerializable*> 4 ({VectorSerializable float64 [
-value SGVector<float64> 2 ({0.65007014456066}{0.6398374522576915})
+value SGVector<float64> 2 ({0.650070144560659}{0.6398374522576915})
 ]}{Serializable float64 [
-value float64 0.1625803785565046
+value float64 -0.1690445308697427
 ]}{Serializable float64 [
-value float64 1
+value float64 0.95
 ]}{VectorSerializable float64 [
-value SGVector<float64> 20 ({1}{1}{-1}{1}{1}{-1}{1}{1}{1}{-1}{-1}{1}{1}{-1}{1}{1}{1}{-1}{-1}{-1})
+value SGVector<float64> 20 ({1}{1}{-1}{1}{-1}{-1}{1}{1}{1}{-1}{-1}{1}{1}{-1}{1}{1}{1}{-1}{-1}{-1})
```

Comment:
so the accuracy drops if we leave the bias of the newton SVM algorithm rather than overwriting it after ....?
@iglesias do you have comments on that
@sonney2k you by any chance?

Reply:
What do leave and overwriting after mean in here? ;-)

Reply:
if compute_bias is true then the bias of newtonsvm is overwritten at the [end of training](https://github.com/shogun-toolbox/shogun/blob/770f748f4fd3b6db66ac08e699924f7152a8557d/src/shogun/machine/LinearMachine.cpp#L171) with the mean diffrence. Here we set it to false so that the bias calculated by newtonsvm is used to predict and we find that accuracy drops.

Reply:
Discussed with Heiko offline. It looks fishy indeed. It might make sense in some case for regression.